### PR TITLE
AT-4491 Remove gaps in full-bleed topper on mobile

### PR DESCRIPTION
**Pull request by user896724**
Remove min-height from topper content and visual to remove potential extra vertical space.  The gap below the visual was pointed out as an error, and I made a similar fix to the content in https://github.com/Financial-Times/o-topper/pull/80, but I'm not sure about it here:

Before:

Selection_006

After:

Selection_007

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-topper bundle size difference from 3.1.3 to 01b3e7248924bea9559591f5f3f3faf64f32c9ee
No significant bundle size differences found.

**Comment by user896724** [comment]
Here it is with just the visual min-height removed:

Selection_008

**Comment by origamiserviceuser** [comment]
o-topper bundle size difference from 3.1.3 to 728e9d5dc86f20753107038c0e59be15560ba42c
No significant bundle size differences found.

File: src/scss/themes/_full-bleed-image.scss [hunk 1]
```diff
@@ -18,11 +18,6 @@
 			flex-direction: column;
 			justify-content: center;
 			padding: 20px;
-			min-height: 350px;
```
Comment (notlee) [inline src/scss/themes/_full-bleed-image.scss:21]: I expect we still want this to keep the headline area of the topper tall on mobile

File: src/scss/themes/_full-bleed-image.scss [hunk 2]
```diff
@@ -18,11 +18,6 @@
...
-		}
-
-		.o-topper__visual {
-			min-height: 350px;
```
Comment (notlee) [inline src/scss/themes/_full-bleed-image.scss:25]: I think it's right to remove this one though. It's unhelpful unless we update the image to fill the visual area which I expect we don't want to do here — cropping images without editorial input is problematic

File: src/scss/themes/_full-bleed-image.scss (hunk 1 above)
Comment (user896724) [reply src/scss/themes/_full-bleed-image.scss:21]: Cool - updated

**Comment by user896724** [comment]
Updated to reflect above screenshot, keeping the gaps in the content

**Comment by origamiserviceuser** [comment]
o-topper bundle size difference from 3.1.3 to 3e7680076d4f3474a5f2746d35737f53b719a182
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-topper bundle size difference from 3.1.3 to 1ef2a438d285d7816baf0fde355e0d569853f9ee
No significant bundle size differences found.

**Review by notlee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v3.1.4 :tada:

**Pull request merged**